src/main/java/org/sagebionetworks/bridge/exporter/worker/ExportWorkerManager.java

```diff
@@ -566,8 +565,8 @@ public void endOfStream(ExportTask task) throws RestartBridgeExporterException {
             } else {
                 redriveTag = REDRIVE_TAG_PREFIX + tag;
             }
-            BridgeExporterRequest redriveRequest = new BridgeExporterRequest.Builder().copyOf(request).withDate(null)
-                    .withStartDateTime(null).withEndDateTime(null).withRecordIdS3Override(filename).withTag(redriveTag)
+            BridgeExporterRequest redriveRequest = new BridgeExporterRequest.Builder().copyOf(request)
+                    .withEndDateTime(null).withExportType(null).withRecordIdS3Override(filename).withTag(redriveTag)
                     .withRedriveCount(redriveCount + 1).build();
```

Comment:
Should specify ignoreLastExportTime just to be clear and safe. Similarly for the redrive table case.